recipe/meta.yaml

```diff
@@ -130,52 +130,55 @@ outputs:
         - fitsverify
         - frozendict >=2.2.0
         - future
+        - galsim
         - getcalspec >=2.0.0
         - git
         - git-lfs
         - h5py
         - healpy
         - healsparse
         - hpgeom >=0.8.2
-        - htcondor
+        - htcondor >=23.0,<23.1
         - humanize
         - idds-client >=0.11.0
         - idds-doma >=0.11.0
         - iminuit >=2
         - lmfit
         - lsstdesc.coord
-        - matplotlib-base !=3.6.0
-        - mkl  # [linux64]
+        - lsst-ts-xml
+        - matplotlib-base !=3.6.0,>=3.8.0
         - moto >=3
         - mpi4py
+        - nest-asyncio
         - networkx
+        - ngmix-core
         - numexpr =2
         - {{ pin_compatible('numpy') }}
         - panda-client !=1.5.65
-        - pandas <2.1
+        - pandas
```

Comment:
pandas has promised some possibly breaking changes for panda=3.  Should we pin it to `=2` here?